api/src/test/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceTest.java

```diff
@@ -619,6 +619,31 @@ public void testUseAbsorb_FalseWhenMoodlePreviouslyUsed() throws Exception {
     assertThat(complianceTrainingService.useAbsorb()).isFalse();
   }
 
+  @Test
+  public void testTrainingsEnabled_TrueWhenMigrationPauseInactive() throws ApiException {
+    setRTTrainingCompletedWithMoodle();
+    providedWorkbenchConfig.moodle.trainingMigrationPauseActive = false;
+
+    assertThat(complianceTrainingService.trainingsEnabled()).isTrue();
+  }
+
+  @Test
+  public void testTrainingsEnabled_TrueWhenMoodleNeverUsed() {
+    providedWorkbenchConfig.moodle.trainingMigrationPauseActive = true;
+    // setRTTrainingCompletedWithMoodle is not called
+
```

Comment:
Is there any way to confirm/test the source of user would be moodle apart from setRTTrainingCompletedWithMoodle is not called?

Reply:
I don't think this test was very clear, so I changed the test name to `testTrainingsEnabled_TrueDuringPauseIfNoTrainingsTaken` and the comment to `// The user has taken no trainings`.

Coupled with the Absorb test below, I think the tests now validate all three cases: no training, Absorb, and Moodle.

Do the changes address your concern?

Reply:
Yes, thanks